src/app.rs

```diff
@@ -147,6 +147,7 @@ pub struct RepoConfig {
     pub suggested_repo_name: Option<String>,
     pub path: PathBuf,
     pub collection_id: Option<String>,
+    pub deploy_collection_id: bool,
```

Comment:
I think you want  #[serde(default)] there too

Reply:
Makes sense, so it will default to false